.github/workflows/README.md

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Hello Naftaly. Could you please rename this in a way it is relevant only to `loadtest.yaml` workflow? Maybe call it `loadtest-README.md` or so?

Reply:
Sure, I'll rename it